gbpservice/neutron/nsf/core/cfg.py

```diff
@@ -0,0 +1,35 @@
+from oslo_config import cfg
+
+OPTS = [
+    cfg.IntOpt(
+        'workers',
+        default=1,
+        help=_('#of workers to create.')
+    ),
+    cfg.StrOpt(
+        'RpcLoadBalancer',
```

Comment:
Can we follow a consistent model. all lower case or all upper case?

Why is this called RpcLoadBalancer? Is it distributing across queues?